nbuild/stdenv/fetch.py

```diff
@@ -35,7 +35,7 @@ def fetch_url(
     if not _check_file(path, md5, sha1, sha256):
         ilog(f"Fetching {url}")
         req = requests.get(url, stream=True)
-        with open(path, 'wb') as file:
+        with open(path, 'wb+') as file:
```

Comment:
The `+` allows reading as well, is it needed here ?

Reply:
I thought the `+` was about creating the file if it doesn't exist. Am i missing something?

Reply:
Lel, stack overflow baited my on this one.
My bad, reverting this in next commit.